base/applications/cmdutils/more/more.c

```diff
@@ -54,6 +208,13 @@ PagePrompt(PCON_PAGER Pager, DWORD Done, DWORD Total)
     HANDLE hInput = ConStreamGetOSHandle(StdIn);
     DWORD dwMode;
     KEY_EVENT_RECORD KeyEvent;
+    BOOL fCtrl;
+    DWORD nLines;
+    WCHAR ch;
+    static UINT s_nPromptID = IDS_CONTINUE_PROGRESS;
+    static WCHAR s_chSubCommand = 0;
+top:
```

Comment:
```suggestion

top:
```
Also, rename "top" into "Restart" or something more understandable / less generic.

Reply:
OK.